alf/environments/alf_environment.py

```diff
@@ -218,6 +218,12 @@ def step(self, action):
         self._current_time_step = self._step(action)
         return self._current_time_step
 
+    def sync_progress(self):
```

Comment:
Adding this since in the call to ``env.sync_progress()`` in ``policy_trainer.py``, ``env`` could be a non-parallel env. Adding this interface to unify the interface without additional codes to do conditional call based on the typr of env.